packages/prices/utils/price.ts

```diff
@@ -73,7 +73,7 @@ export const getCurrencyFromPriceResponse = (
 	// Currency data object, for example an API response containing currency formatting data.
 	currencyData?:
 		| CurrencyResponse
-		| Record< string, never >
+		| Record< string, never | unknown >
```

Comment:
This part had been changed to prevent a TS error related to the `prices` attribute in the following section:
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L51